ddtrace/llmobs/_integrations/mcp.py

```diff
@@ -99,3 +105,30 @@ def _llmobs_set_tags_server(self, span: Span, args: List[Any], kwargs: Dict[str,
             output_value = safe_json(response)
 
         span._set_ctx_item(OUTPUT_VALUE, output_value)
+
+    def _llmobs_set_tags_initialize(self, span: Span, args: List[Any], kwargs: Dict[str, Any], response: Any) -> None:
+        span._set_ctx_items({NAME: "MCP Client Initialize", SPAN_KIND: "task", OUTPUT_VALUE: safe_json(response)})
```

Comment:
just checking in - are we happy with span names with spaces instead of underscore/dashes?

Reply:
yeah good question, i think i did it like this bc we were already doing this for the other methods we already instrument. i guess it's a question of our product experience as a whole, and then changing would be kinda breaking i guess.

i guess for mcp it's best to keep it consistent - but fine to do it either way. if we do go with `.` and `_` for names, we can't change the existing mcp names as it would be breaking for evaluations sampling i think (or, maybe not? do they apply to tool spans?)

Reply:
Cool, as long as we are consistent I'm happy with this